lib/matplotlib/axes/_axes.py

```diff
@@ -1141,6 +1141,11 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
         if len(positions) == 0:
             return []
 
+        # remove 'singular' keys from **kwargs dict in order to prevent them
+        # being lazily passed as LineCollection props, where they override the
+        # effect of the 'plural' versions (e.g. 'color' overrides 'colors')
+        cbook.strip_problem_kwargs(kwargs, 'color', 'linewidth', 'linestyle')
```

Comment:
Yes, I thought that was your intent, based on your suggestion to add a warning to the effect of `warnings.warn("'color' kwarg ignored")`